lib/rules/selector-disallowed-list/README.md

```diff
@@ -66,3 +66,49 @@ a
 ```css
 a[href] {}
 ```
+
+## Optional secondary options
+
+### `iterateThroughList: true | false` (default: `false`)
+
+Iterate through each individual selector in a selector list.
+
+For example, with `true`.
+
+Given:
+
+```json
+[".foo"]
+```
+
+The following pattern is considered a problem:
+
+<!-- prettier-ignore -->
+```css
+.bar, .foo {}
+```
+
+The following pattern is _not_ considered a problem:
+
+<!-- prettier-ignore -->
+```css
+.bar .foo {}
+
+###  `ignore: ["inside-block"]`
```

Comment:
There are some typos.

````suggestion
.bar .foo {}
```

### `ignore: ["inside-block"]`
````

Reply:
Eek , I'll do another pass through of the README